internal/sdk/serverlesscloudgateways.go

```diff
@@ -26,6 +26,9 @@ func newServerlessCloudGateways(sdkConfig sdkConfiguration) *ServerlessCloudGate
 }
 
 // CreateServerlessCloudGateway - Create a new serverless cloud gateway
+// **Pre-release Endpoint**
```

Comment:
This is unexpected. Let me investigate the source specs

Reply:
Ah! This is expected. This API doesn't actually work any more. We used the beta API before we had the idea for a beta provider.

As the API is gone, we should remove the resource. We can do it in a minor version as although it's a breaking change, it's a beta and the API no longer works anyway. We can follow up outside of this PR